Source/JavaScriptCore/runtime/ProgramExecutable.cpp

```diff
@@ -112,7 +112,7 @@ JSObject* ProgramExecutable::initializeGlobalProperties(VM& vm, JSGlobalObject*
             RETURN_IF_EXCEPTION(throwScope, nullptr);
             switch (status) {
             case GlobalPropertyLookUpStatus::NonConfigurable:
-                return createSyntaxError(globalObject, makeString("Can't create duplicate variable that shadows a global property: '", String(entry.key.get()), "'"));
+                return createSyntaxError(globalObject, makeString("Can't create duplicate variable that shadows a global property: '"_s, String(entry.key.get()), '\''));
```

Comment:
Should be able to pass `entry.key` without `String` to avoid reference count churn.